manifests/t/tumic0/GPXSee/13.8/tumic0.GPXSee.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: tumic0.GPXSee
+PackageVersion: "13.8"
+PackageLocale: en-US
+Publisher: Martin Tůma
+PublisherUrl: https://www.gpxsee.org
+PublisherSupportUrl: https://github.com/tumic0/GPXSee/issues
+# PrivacyUrl:
+Author: tumic0
+PackageName: GPXSee
+PackageUrl: https://www.gpxsee.org
+License: GNU General Public License version 3.0 (GPLv3)
+LicenseUrl: https://github.com/tumic0/GPXSee/blob/master/licence.txt
+Copyright: Copyright (C) 2023 tumic0
```

Comment:
"Martin Tůma" should IMHO go here instead, makes no sense to use the GitHub login here...

Reply:
Can you make a code suggestion using the suggestion feature? I'm not able to make changes from my phone, but I can apply code review suggestions. Otherwise, it will be a few days before I get to this